client/sql/dataframereader.go

```diff
@@ -0,0 +1,41 @@
+package sql
+
+import proto "github.com/apache/spark-connect-go/v_3_4/internal/generated"
+
+type DataFrameReader interface {
+	Format(source string) DataFrameReader
+	Load(path string) (DataFrame, error)
```

Comment:
I think it's fine to create incremental PRs for the new features, but it would be great to leave a comment here showing which methods are missing in the interface.

Reply:
Good suggestion, added TODO for missing methods.